### PR TITLE
Created receipt controller

### PR DESCRIPTION
**Pull request by isabelt220**

### CODE REVIEW

File: server/controller/receipt_controller.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from datetime import date
+
+class ReceiptController:
```
Comment (bonnieli) [inline server/controller/receipt_controller.py]: this looks like a model, rather a controller! 

other than that, this looks like it's on the right track to be a good Receipt model :)

Comment (ethanlow23) [reply server/controller/receipt_controller.py]: I think for the Models, we still need to map the receipt class to a table for the database. But the columns you created looks good! Don't forget the primary key!

Comment (isabelt220) [reply server/controller/receipt_controller.py]: I made some changes to Receipt controller and created Receipt model that handles the actual creation of the receipt. Is this more in the right direction? Also, I haven't added the primary key yet but I'll get to it later today!

**Comment by ethanlow23** [comment]
Looks pretty good! I do have a question about the create route though. Does empty string work? Don't forget to check if there was an error in creating the receipt and also returning success/fail data for the frontend. For the database, don't forget its going to belong to a user so we need to be able to make joins. We also do not need to take data from user for date created because it will automatically take the current date.

**Review by bonnieli — APPROVED** [review]

**Review by ethanlow23 — APPROVED** [review]

**Pull request merged**